source/write/replace.txt

```diff
@@ -0,0 +1,283 @@
+       | Defaults to ``false``.
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``comment``
+     - | Specifies a comment of any valid BSON type to attach to the operation. 
+         Once set, this comment appears alongside records of this command in the following locations:
+
+       - :ref:`mongod log messages <log-messages-ref>`, in the
+         ``attr.command.cursor.comment`` field
+       - :ref:`Database profiler output <profiler>`, in the :data:`system.profile.command <system.profile.command>` ``comment`` field
+       - :dbcommand:`currentOp` output, in the :data:`currentOp.command
+         <currentOp.command>` ``comment`` field
+
+       | For more information, see the :manual:`insert Command
+         Fields guide </reference/command/insert/#command-fields>` in the {+mdb-server+} manual.
+
+   * - ``hint``
+     - | Specifies the index to scan for documents that match the query filter.
+         For more information, see the :manual:`hint field </reference/command/update/#std-label-update-command-hint>`
+         in the {+mdb-server+} manual. 
+
+   * - ``let``
+     - | Specifies a document containing variables and their values to be used in the ``replace_one()`` method. 
+         This allows you to improve code readability by separating the variables from the operation text. 
+         Values must be constant or closed expressions that don't reference document fields. 
+         For more information, see the 
+         :manual:`update Command Fields guide </reference/command/update/#std-label-replace-let-syntax>`
+         in the {+mdb-server+} manual.
+
+   * - ``upsert``
+     - | Specifies whether the replace operation performs an upsert operation if no 
+         documents match the query filter.
+       | Defaults to ``false``.
+
+   * - ``write_concern``
+     - | Sets the write concern for the operation.
+         For more information, see :manual:`Write Concern </reference/write-concern/>`
+         in the {+mdb-server+} manual.
+
+Example: hint
+^^^^^^^^^^^^^^^^^
+
+The following example uses the ``create_index()`` method to create an ascending single-field index 
+on the ``name`` field. It then passes a ``mongocxx::options::replace`` object to the ``replace_one()``
+method after setting its ``hint`` field to the new index. This instructs the replace operation to search the ``name`` 
+field index when replacing a document that has a ``name`` field value of ``"Nobu"``:
+
+.. literalinclude:: /includes/write/replace.cpp
+   :start-after: start-replace-options-hint
+   :end-before: end-replace-options-hint
+   :language: cpp
+   :dedent:
+
+To learn more about indexes, see :ref:`Optimize Queries with Indexes <cpp-indexes>`.
+
+Example: upsert 
+^^^^^^^^^^^^^^^^^^^
```

Comment:
S: I'd clarify that this is an option in the header. Also, make sure your header underline is the same length as the header text
```suggestion
Example: upsert Option
^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Applies to the hint example as well